Shared/Mobile-Rounds.ViewModels/Regular/Region/RegionModel.cs

```diff
@@ -0,0 +1,25 @@
+﻿using Mobile_Rounds.ViewModels.Shared;
```

Comment:
I also suggest we keep the actual model class separate from this one. Mostly because this class is UI specific, while the other one is the server-mobile contract.